Monika After Story/game/script-topics.rpy

```diff
@@ -16165,3 +16165,54 @@ label monika_giving_criticism:
     m 3dsu "Graciously accept you can't change everyone's mind and stay considerate when assessing someone else's work."
     m 3hub "...That'd be Monika's Critique Tip of the Day, ahaha!"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_curse_words",
+            category=["advice", "life"],
+            prompt="Curse words",
+            random=True,
+            pool=False
+        )
+    )
+
+default persistent._mas_pm_swear_frequency = None
+
+label monika_curse_words:
+    m 3etd "Say, {w=0.2}[player], {w=0.2}do you ever swear?"
+    menu:
+        m "Say, [player], do you ever swear?{fast}"
```

Comment:
I feel like we're asking the wrong question given what the variable stores. Imo this should ask `Do you swear often?`

A blanket `yes` doesn't mean that you swear often, it just means you do indeed swear.

Reply:
```suggestion
        m "Say, [player], do you swear often?{fast}"
```